script.py

```diff
@@ -0,0 +1,20 @@
+import csv
+from cpovc_main.models import SetupList
+with open('/home/richie/Desktop/fixtures3 - Sheet1.csv') as csv_file:
```

Comment:
If this script is to work on the server the path should be relative. using os.path.join() will be best. you can use the one on settings.py file for BASE_DIR as an example

Reply:
I have removed the scripts.py file